examples/computability/lambda/churchnumScript.sml

```diff
@@ -952,3 +935,4 @@ Proof
 QED
 
 val _ = export_theory()
+val _ = html_theory "churchnum";
```

Comment:
If you cause HTML files to be generated as part of script-file running, can you also clean them up by adding the relevant names to an `EXTRA_CLEANS` line in the directory's `Holmakefile` please?

Reply:
I just modified that `EXTRA_CLEANS`. I've already done such changes in `Holmakefile`s in sub-directories of `examples/lambda`.